pandas/tests/series/methods/test_append.py

```diff
@@ -61,15 +61,15 @@ def test_append_tuples(self):
 
         tm.assert_series_equal(expected, result)
 
-    def test_append_dataframe_regression(self):
-        # GH 30975
-        df = pd.DataFrame({"A": [1, 2]})
-        result = df.A.append([df])
-        expected = pd.DataFrame(
-            {0: [1.0, 2.0, None, None], "A": [None, None, 1.0, 2.0]}, index=[0, 1, 0, 1]
```

Comment:
is there anything about this old test that was correct that should be retained?

Reply:
I don't think so. That case is testing for the successful appendage of a Series and a DataFrame. Which is exactly what we're trying to raise an error for. So I guess there's nothing right about the case.
What do you think about this @jbrockmendel 

Thanks!